### PR TITLE
Feature/webhooks

### PR DESCRIPTION
**Pull request by guymatz**
Resolves Issue #272

### CODE REVIEW

**Comment by guymatz** [comment]
@ssbarnea The error I', getting is:
IOError: [Errno 2] No such file or directory: 'ChangeLog'
error in setup command: Error parsing /home/travis/build/pycontribs/jira/setup.cfg: IOError: [Errno 2] No such file or directory: 'ChangeLog'

I don't think this has anything to do with my code . . .  Any ideas about how I can fix this?  Thanks!

File: jira/resources.py [hunk 1]
```diff
@@ -790,6 +792,29 @@ def __eq__(self, other):
         return self.id == other.id and self.name == other.name
 
 
+class Webhook(Resource):
+    """A JIRA webhook."""
+
+    WEBHOOK_BASE_URL = '{server}/rest/webhooks/{agile_rest_api_version}/{path}'
```
Comment (hdost) [inline jira/resources.py]: You seem to define the webhook url in two places. Here and line 223

**Comment by hdost** [comment]
@guymatz could you rebase off of develop?

**Comment by guymatz** [comment]
Happy to, but I'm not *that* comfortable with git . . .  got any
instructions?

**Comment by hdost** [comment]
I did it in #566 , for this case it didn't actually do a rebase. 
Essential commands:
```
git checkout develop # This it to get our correct base branch
git pull --rebase # This is do update
git checkout -b refactored-branch # Creates branch from develop
git cherry-pick <COMMIT IDS SEPARATED BY SPACES> # To find these commits just look at which ones are your changes
git branch -D <old branch>
git branch -m <old branch name> # This renames your current branch
git push
```
Those commands assume you're working from the pycontribs/jira as your "origin" remote. If not, then to get up to date you would need to do something like the following.
```
git remote add upstream git://github.com/pycontribs/jira.git
git fetch upstream
git checkout develop
git rebase upstream develop
```
Pick up from the new branch creation.

**Comment by guymatz** [comment]
Thanks!  I'm using the second method in which I add pycontribs/jira.git as
upstream, however I am getting this error:
$ git rebase upstream develop
fatal: Needed a single revision
invalid upstream 'upstream'

Shall I follow the advice here?
https://stackoverflow.com/questions/4798080/git-rebase-fatal-needed-a-single-revision

Thanks,
Guy

**Pull request closed without merging**